### PR TITLE
Added basic help page

### PR DESCRIPTION
**Pull request by psiinon**
This is as much to test the new help generator code as anything else, so its worth checking the help structure as much as the content.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<TITLE>
+Token generation and analysis
+</TITLE>
+</HEAD>
+<BODY BGCOLOR="#ffffff">
+<H1>Token generation and analysis</H1>
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html]: Same here

File: src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+This add-on allows you to generate and analyse pseudo random tokens, such as those used for session handling or CSRF protection
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html]: Period?

File: src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<p>
+To generate and analyse tokens:
+<ul>
+<li>Find a request that contains the token you want to generate</li>
+<li>Right click in the Sites tab or one of the bottom tabs and select 'Generate tokens...'</li>
+<li>Choose the token you want to generate</li>
+<li>Press the Generate button</li>
+<li>The responses and tokens will then be listed in the 'Token Gen' tab</li>
+<li>When all of the tokens have been generated the 'Analyse Tokens' dialog will be displayed.</li>
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html]: Drop the period here? Or add them to the other bullets?

**Comment by psiinon** [comment]
Fixed those comments.

File: src/org/zaproxy/zap/extension/tokengen/ZapAddOn.xml [hunk 4]
```diff
@@ -1,13 +1,13 @@
 <zapaddon>
 	<name>Token generation and analysis</name>
-	<version>11</version>
+	<version>12</version>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/tokengen/ZapAddOn.xml]: Version number should be kept at 11 (it was not released yet).

File: src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/contents/tokengen.html]: To show properly in Java help, for all languages, it should be defined the charset here (`<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">`).

File: src/org/zaproxy/zap/extension/tokengen/resources/help/index.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE index
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Index Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/index_2_0.dtd">
+
+<index version="2.0">
+    <!-- index entries are merged (sorted) into core index -->
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/index.xml]: Could be removed.

File: src/org/zaproxy/zap/extension/tokengen/resources/help/map.jhm [hunk 7]
```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+	<mapID target="tokengen-icon" url="contents/images/barcode.png" />
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/tokengen/resources/help/map.jhm]: Could have the same indentation as the following element.

**Comment by psiinon** [comment]
Changes made :)

**Comment by thc202** [comment]
Thank you!

**Pull request merged**